src/openms/include/OpenMS/METADATA/Precursor.h

```diff
@@ -60,7 +60,24 @@ namespace OpenMS
   {
 
 public:
-    // RULE of 0 for Ctor, Dtor, ...
+    /// Constructor
+    Precursor();
```

Comment:
= default ?

Reply:
no strong feelings here, since its either inlining it (potentially), thus making the binaries larger; or not...

Same with dtor...

Reply:
Given that there is not much difference except of more concise code I am for defaulting as much as possible.